src/xunit.performance.nuspec

```diff
@@ -54,9 +54,7 @@
         <dependency id="System.Text.Encoding" version="4.0.10" />
         <dependency id="System.Threading" version="4.0.10" />
         <dependency id="System.Threading.Tasks" version="4.0.10" />
-        <dependency id="xunit.abstractions" version="2.0.0" />
-        <dependency id="xunit.extensibility.core" version="2.1.0" />
-        <dependency id="xunit.extensibility.execution" version="2.1.0" />
+        <dependency id="Microsoft.DotNet.BuildTools.TestSuite" version="1.0.0-prerelease-508-01" />
```

Comment:
To add a bit of context - I get concerned when we have to re-package because it means that there is a chance that dependencies change in some fashion (libraries, tfm, etc.)  CLI is different because we run under dotnet test.